src/helpers/authentication.ts

```diff
@@ -0,0 +1,6 @@
+const validUIAccess = (req, res, next): void => {
+  if (!req.user) res.redirect('/login')
```

Comment:
`return res.redirect`

without return, you may be trying to send back multiple responses for one request, resulting in warnings.